source/renderer/app/utils/transaction.js

```diff
@@ -76,6 +76,7 @@ export const isTransactionAmountInFilterRange = (
   transaction: WalletTransaction
 ) => {
   const { amount } = transaction;
+  const amountAbsoluteValue = amount ? amount.absoluteValue() : null;
```

Comment:
@DeeJayElly please improve this. All amounts **need** to be in BigNumber format.

Reply:
Fixed